src/getbmiss.f

```diff
@@ -23,7 +23,7 @@
 C>
       REAL*8 FUNCTION GETBMISS()
 
-      INCLUDE 'bufrlib.inc'
+      USE MODV_BMISS
```

Comment:
I know we relax to doing `USE MODULE_NAME`.
But, I would recommend changing that habit to:
`USE MODULE_NAME, ONLY: USED_ATTRIBUTE_FROM_THE_MODULE`

Reply:
Also, should the filenames be changed to *.F90? Now that you are using F90 language features, .f files will confuse some tools (doxygen?) because the expectation is that .f files contain F77 code and .F90 files contain F90 code...

Reply:
I never went through the entire code and changed everything to f90; rather, I had only changed the bfrini.f routine to bfrini.f90 during the previous PR, so that I didn't have to worry about the value of MASTER_TABLE_DIR being longer than 72 characters.

I agree that it would be best to use f90 going forward, and I can start the habit of doing that for any new source files (and I'll also read up some more on ONLY: USED_ATTRIBUTE_FROM_THE_MODULE, which I'd never seen before ;-)  But it would be a major undertaking to modify more than 300 existing source files to change all comments, continuation line punctuation, etc. to use f90 syntax.  For now, all current .f files are indeed F77, and all current .f90 files are indeed F90, so everything is well-defined to Doxygen.

FWIW, I really do appreciate all of the bits of help and advice to move this library forward into a more modern software development and maintenance paradigm.  I've really learned a lot from all of you re: CMake, Doxygen, interfacing C and Fortran, etc. and these will serve me well going forward.  But I don't see a lot of bang for the buck in going through all of the existing code and converting it to f90 just for the sake of f77 vs. f90.  If there's a reason to do so for improved functionality (such as was the case for bfrini.f) then that's one thing, but again I don't see much reason at this point to do so for the entire library.

Reply:
I meant to suggest using the `use module, only: attribute` in files that are touched going forward.
There is no need at this time to go through 300+ files to do that in one go, so that in a few months some or many will have been updated and make the move easier.  That's just my opinion.